schema.sql

```diff
@@ -0,0 +1,35 @@
+DROP DATABASE IF EXISTS rewhy;
+
+CREATE DATABASE rewhy;
+
+USE rewhy;
+
+DROP TABLE IF EXISTS types, categories, recommended;
```

Comment:
This is potentially unnecessary if you are dropping the database any time you run the file.